appveyor.yml

```diff
@@ -27,6 +27,11 @@ environment:
       PYTHON_ARCH: "64"
       NOX_SESSION: "test-3.7"
 
+    - PYTHON: "C:\\Python38-x64"
+      PYTHON_VERSION: "3.8.x"
```

Comment:
And this one does not work yet because Tornado 6 does not support the proactor event loop which is the default on Windows + 3.8 so we need to configure the event loop differently here.

Reply:
Opened https://github.com/urllib3/urllib3/pull/1777 for this one!